spec/cohort-spec.js

```diff
@@ -67,13 +67,14 @@ describe('Manager', () => {
       '001'
     )
     cohort.addStudent(student1)
-    cohort.removeStudentById('01')
-    expect(cohort.students.includes(student1)).toBe(true)
+    cohort.removeStudentById('001')
+    expect(cohort.students.includes(student1)).toBe(false)
```

Comment:
could swap .toBe(false) with .toBeFalse() if you wanted to (not that it'd make much of a difference, I reckon)